go/oprf/groups/ecgroup/ec.go

```diff
@@ -349,14 +349,12 @@ func (p Point) decompress(curve GroupCurve, buf []byte) (Point, error) {
 		x3Plusax := new(big.Int).Mul(x2Plusa, x)
 		x3PlusaxPlusb := new(big.Int).Add(x3Plusax, curve.ops.Params().B)
 		y2 = new(big.Int).Mod(x3PlusaxPlusb, order)
-		break
```

Comment:
Why remove these? Did the linter flag them as redundant?

Reply:
yes, they are redundant, because Go's default is to break after a case